### ISSUE TITLE
Testing whether 'v == ic', should be whether 'v is ic'

### ISSUE THREAD

**Issue by osterm38** [gruns/icecream#15]
When trying to run ic(x) on a numpy.ndarray object, I get an error saying 'the truth value of an array with more than one element is ambiguous', pointing back to the function determinePossibleIcNames.  I think a quick fix for the this could be changing the test 'v == ic' to 'v is ic', since the latter tests to see if v and ic point to the same (IceCreamDebugger) object, while the former calls the __eq__ function which is causing the issue.

**Comment by gruns** [comment]
Wonderful catch.

Changing

```python
    localNames = [name for name, v in callFrame.f_locals.items() if v == ic]
    globalNames = [name for name, v in callFrame.f_globals.items() if v == ic]
```

to

```python
    localNames = [name for name, v in callFrame.f_locals.items() if v is ic]
    globalNames = [name for name, v in callFrame.f_globals.items() if v is ic]
```

indeed fixes this bug.

This fix will be included in the next version of icecream.

**Comment by gruns** [comment]
Fixed in v1.2. Upgrade with

```
pip install icecream --upgrade
```

Thank you again for bringing this issue to my attention @osterm38.
